### PR TITLE
NOISSUE - Update dependencies

### PR DESCRIPTION
**Pull request by dborovcanin**
### What does this do?
This pull request updates dependencies to a more recent version.
It also fixes a messed params order bug in Timescale Reader main.go file.

### Which issue(s) does this PR fix/relate to?
There is no such issue.

### List any changes that modify/break current functionality
There are no such changes.

### Have you included tests for your changes?
No new tests were added, but I also manually tested the basic flow (CRUD ops on Things and Users, and messaging).

### Did you document any new/modified functionality?
N/A

### Notes
N/A

### CODE REVIEW

File: cmd/timescale-reader/main.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func main() {
 
 	repo := newService(db, logger)
 
-	auth, authHandler, err := authClient.Setup(envPrefix, svcName, cfg.JaegerURL)
+	auth, authHandler, err := authClient.Setup(envPrefix, cfg.JaegerURL, svcName)
```
Comment (SammyOina) [inline cmd/timescale-reader/main.go]: this addressed here https://github.com/mainflux/mainflux/pull/1834/files#diff-333e96fd1321194ba79c77c22f58e2746d699543d516fa8697006dcd39c57902 , could result in duplicates/conflicts

Comment (dborovcanin) [reply cmd/timescale-reader/main.go]: Reverted.

**Review by drasko — APPROVED** [review]
LGTM

**Comment by fbugarski** [comment]
LGTM

**Pull request merged**